scripts/world_bank/datasets/datasets.py

```diff
@@ -0,0 +1,240 @@
+# Copyright 2021 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Populates a list of downloadable world bank datasets.
+
+It can also optionally download these datasets.
+
+To run this script:
+
+python3 datasets.py
+
+Be default, the script will only produce a CSV with the list of datasets.
+Once the CSV is created, to download the datasets, the download_datasets method will need to be called explicitly.
+
```

Comment:
Can you also mention where files would be downloaded and what the output folder will contain?

Reply:
Done.